packages/aws-cdk-lib/package.json

```diff
@@ -534,6 +534,7 @@
     "./aws-medialive": "./aws-medialive/index.js",
     "./aws-mediapackage": "./aws-mediapackage/index.js",
     "./aws-mediastore": "./aws-mediastore/index.js",
+    "./aws-mediatailor": "./aws-mediatailor/index.js",
```

Comment:
Why is this being added here?

Reply:
The build will fail without adding this line.

Reply:
Curious, how is this related to your change? How did you discover it was missing?